botorch/models/gp_regression_fidelity.py

```diff
@@ -179,47 +166,11 @@ def construct_inputs(
         return inputs
 
 
-class FixedNoiseMultiFidelityGP(SingleTaskMultiFidelityGP):
```

Comment:
this removal would also be good to mention in the title and/or summary

Reply:
Oh I bet this is a rebase issue and this change was introduced in a separate PR.